messages/src/lib.rs

```diff
@@ -926,6 +929,40 @@ impl Decode for HpkeConfig {
     }
 }
 
+/// DAP protocol message representing a list of HPKE configurations.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct HpkeConfigList(Vec<HpkeConfig>);
+
+impl HpkeConfigList {
+    /// The media type associated with this protocol message.
+    pub const MEDIA_TYPE: &'static str = "application/dap-hpke-config-list";
+
+    /// Construct an HPKE configuration list.
+    pub fn new(hpke_configs: Vec<HpkeConfig>) -> Self {
+        Self(hpke_configs)
+    }
+
+    pub fn hpke_configs(&self) -> &[HpkeConfig] {
+        &self.0
+    }
+
+    pub fn as_vec(self) -> Vec<HpkeConfig> {
```

Comment:
nit: I think this can be removed once the other comment is taken.

Reply:
Per typical [naming conventions](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv), `into_vec()` would be better for this method.